.github/workflows/integration-tests.yaml

```diff
@@ -295,3 +295,49 @@ jobs:
         with:
           name: tests-logs
           path: artifacts
+
+  integration-tests-legacy-v2:
+    name: Integration Tests Legacy V2
+    runs-on: ubuntu-20.04
+    needs: prepare-integration-tests
+    steps:
+      - name: Apicurio Registry (Version 2.5.x) Checkout
```

Comment:
This step seems oddly complicated, it might become something like:

```yaml
- name: Checkout Registry 2.5
  uses: actions/checkout@v4
  with:
    ref: '2.5.x'
    path: registry-v2
 ```

Reply:
Ah yes - this was a copy/paste job.  I like this better and will change it.